airflow/providers/docker/operators/docker.py

```diff
@@ -317,18 +317,13 @@ def _run_image_with_mounts(
             if self.retrieve_output:
                 return self._attempt_to_retrieve_result()
             elif self.do_xcom_push:
-                log_parameters = {
-                    'container': self.container['Id'],
-                    'stdout': True,
-                    'stderr': True,
-                    'stream': True,
-                }
+                if len(log_lines) == 0:
+                    return None
                 try:
                     if self.xcom_all:
-                        return [stringify(line).strip() for line in self.cli.logs(**log_parameters)]
+                        return [stringify(line).strip() for line in log_lines]
```

Comment:
stringify and strip is already done in `log_lines`
```suggestion
                        return log_lines
```

Reply:
Fixed